frontend/src/metabase/visualizations/echarts/pie/constants.ts

```diff
@@ -36,8 +36,8 @@ export const OTHER_SLICE_MIN_PERCENTAGE = 0.005;
 
 export const OTHER_SLICE_KEY = `${NULL_CHAR}___OTHER___`;
 
-export const OTHER_SLICE_NAME = t`Other`;
+export const getSliceName = () => t`Other`;
```

Comment:
these 2 variables need to be changed to a function since they're not inside an object like the other 2 files. But they're used only in a number of places, so I'm fine changing it like this.